k8s/stage/ingress/ingress.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    kubernetes.io/ingress.class: nginx
```

Comment:
this annotation dictates which load balancer (ingressController, ELB) will serve your application. By "nginx" only, I think it is referencing a cluster shared load balancer (if there's one).
In case we would want to have 1 ELB per application & environment, you should change it to something like 'nginx-refractr-stage' [1], and deploy a IngressController in this same 'refractr-stage'[2].

Here an example of ingressController per namespace in Moderator:
[1] https://github.com/mozilla/mozmoderator/blob/master/kubernetes/stage/ingress.yaml#L7
[2] https://github.com/mozilla/mozmoderator/blob/master/kubernetes/stage/ingress-controller.yaml